packages/react-core/src/components/Spinner/examples/Spinner.md

```diff
@@ -20,9 +20,18 @@ import React from 'react';
 import { Spinner } from '@patternfly/react-core';
 
 <React.Fragment>
-  <Spinner size="sm"/>
-  <Spinner size="md"/>
-  <Spinner size="lg"/>
-  <Spinner size="xl"/>
+  <Spinner isSVG size="sm"/>
+  <Spinner isSVG size="md"/>
+  <Spinner isSVG size="lg"/>
+  <Spinner isSVG size="xl"/>
 </React.Fragment>
 ```
+
+### Custom size (SVG only)
```

Comment:
based on @mcoker 's comment, you can remove the `(SVG only)` text.

Reply:
Done.